### DISCUSSION TITLE
Slim EMU SFRC2DA:Error 34

### DISCUSSION THREAD

**Discussion by pdieng** [keirf/flashfloppy#535] [Q&A]
I was looking at a post in the issue section by dchauran on May 11th because I am trying to do exactly what they are doing.  I also have a Tektronix TDS640A but I can't get it to work.
I have Flash Floppy firmware on my SFRC2DA and removed RA, RB and RC. However when ever I try to plug in a thumb drive, it gives me error 34. I have a freshly formatted thumbdrive with just FF.CFG and IMG.CFG. When I plug it in it also creates the empty IMAGE_A.CFG file.

It might also have to do with the error 34, but also the o-scope gives me "File System Media is Missing" as an error if I try to do a save.

Does anyone have an idea of what I can try?

**Comment by keirf** [comment]
Error 34 means no image file found. Probably you have not placed IMG image files on your USB drive?

**Reply by pdieng** [reply]
That's what doesn't make sense to me.
I started with a clean FAT32 formatted 2GB USB drive with just FF.CFG and IMG.CFG.  After I plugged it in, I get error 34 and I have the additional file: IMAGE_A.CFG.

I grabbed those CFG files from the flashfloppy-v3.28\examples folder that I downloaded from the repository.

**Reply by keirf** [reply]
Okay but you need at least one floppy image file. Otherwise FlashFloppy has no "disk" to serve up to your Tektronix host!

**Reply by pdieng** [reply]
Ahhh!

I grabbed my floppy drive, a blank floppy, and made an image of that blank floppy disk using WinHex.  Now I am able to save screen shots to that image on my USB drive!!

Right now I have an unregistered version of WinImage that I used to access the screen shots on the floppy image.  Is there a better way to get to the files or is there a better way to do this?

**Reply by keirf** [reply]
I believe some archival tools like WinZip and 7Zip may be able to open IMG files.

**Reply by pdieng** [reply]
Yes, I am able to extract with 7zip.

Thanks for all of your help!!
